tests/AjaxCallActionsWithDynamicConsent/appsettings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "AzureAd": {
+        "Instance": "https://login.microsoftonline.com/",
+        "Domain": "xxx.onmicrosoft.com",
+        "TenantId": "xxx",
+        "ClientId": "xxx",
+        "ClientSecret": "xxx",
+        "CallbackPath": "/signin-oidc"
```

Comment:
cool. thanks for removing all this. was going to ask :)

Reply:
Reminded myself but forgot about it. I have deleted the App registration on Azure anyway.